src/index.ts

```diff
@@ -48,7 +49,7 @@ window.addEventListener('load', () => {
     (e.target as IDBRequest).result.createObjectStore(objStoreName);
   });
 
-  fileInput.addEventListener('change', (e) => onFileInputChange(e, dbOpenRequest, imgElement, shareBtn));
+  fileInput.addEventListener('fileselected', (e) => onFileInputChange(e as CustomEvent, dbOpenRequest, imgElement, shareBtn));
```

Comment:
not sure if this is valid, but I think it would be slightly cleaner like:

```
(e: CustomEvent) => onFileInputChange(e, ...)
```

Reply:
There's a way, but honestly I'm not sure how. It involves typing the `fileselected` custom event, specific to the `LlamaSelectFab` element.